conans/model/settings.py

```diff
@@ -302,23 +302,24 @@ def values_list(self):
     def items(self):
         return self.values_list
 
-    def update_values(self, vals):
-        """ receives a list of tuples (compiler.version, value)
-        This is more an updated than a setter
+    def update_values(self, values, raise_error=True):
```

Comment:
I think I'd call it something like ``validate=True``, a bit more meaningful

Reply:
what about `raise_undefined=True`?